contributor/index.html

```diff
@@ -14,40 +14,14 @@
-<li>Software Submitter acknowledges the applicability of and agrees to abide by the terms of the Ecma Code of Conduct in Patent Matters with respect to patents that are required to implement the subject final Ecma International Standard.<br>
-</li>
-<li> Except as expressly set forth herein, the Software Submitter reserves for itself all other intellectual property rights in its Software submission and makes no assignment, license or other transfer or any other intellectual property rights.<br>
-</li>
-</ol>
-        </section>
-      
-        
-        <h4>Attachment 1: BSD License</h4>
-        <p>Replace YEAR and OWNER with appropriate values.</p>
-        <pre style="white-space:pre-wrap;">
-Copyright (c) &lt; YEAR &gt;, &lt; OWNER &gt; All rights reserved. Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
-
-1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
-2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
-3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.  
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.  
-        </pre>
+<p>This form is used to grant a license to intellectual property rights related to contributions to ECMAScript standards ECMA-262 and ECMA-402, developed in the Ecma TC39 Royalty-Free Task Group. It is required in the following cases:</p>
+<ul>
+  <li>To attend a TC39 meeting as an invited expert and participate in committee discussion</li>
+  <li>To contribute a normative change to an ECMA-262, ECMA-402 or a proposal repository under the tc39 organization</li>
+</ul>
+<p>This form does not grant anyone the right to contribute to TC39 work in either of those ways.</p>
```

Comment:
"either of those ways"? I'm confused at what this line is saying.

Reply:
I meant, signing the form doesn't mean you have the right to go to a meeting, or mean that your PR will land. Maybe the clarification is especially important about meeting attendance, as it's kind of obvious that there is code review.

Reply:
What do you think of the new text there?